pkg/authorization/apiserver/admission/restrictusers/restrictusers.go

```diff
@@ -134,6 +135,10 @@ func (q *restrictUsersAdmission) Validate(a admission.Attributes) (err error) {
 		return nil
 	}
 
+	if !q.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {
```

Comment:
this should be right before the `rbrLister.List` call - the rest of the checks do not need RBRs to be functional

Reply:
done